radix-engine/src/kernel/call_frame.rs

```diff
@@ -651,10 +651,6 @@ impl<C, L: Clone> CallFrame<C, L> {
         }
 
         for node_id in message.copy_stable_transient_references {
-            if from.depth >= to.depth {
```

Comment:
I remember discussing this before a few months ago - but are we comfortable this check can be removed without issue?

Reply:
Yes, this was mostly a sanity check